medusa/db.py

```diff
@@ -132,6 +132,18 @@ def _execute(self, query, args=None, fetchall=False, fetchone=False):
             logger.log(u'DB error: {0!r}'.format(e), logger.ERROR)
             raise
 
+    def normalizePython2Python3(self, sql_results):
```

Comment:
I don't think it's necessary.
If you want to have all the results in dictionary format, you should modify `_set_row_factory` instead.

Reply:
... or use `row_type='dict'` when initializing `DBConnection`.

Reply:
Definitely more clean. I updated the PR. Thank you.